src/components/lazy-load-css.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { Helmet } from "react-helmet"
+
+// Technique copied from https://web.dev/defer-non-critical-css/#optimize
+
+interface Props {
```

Comment:
nit: `LazyLoadCssProps`

Reply:
Haha again I always just call the props for the default exported component `Props`.